upload/admin/controller/localisation/country.php

```diff
@@ -216,6 +289,7 @@ protected function getList() {
 		$data['sort_name'] = $this->url->link('localisation/country', 'user_token=' . $this->session->data['user_token'] . '&sort=name' . $url);
 		$data['sort_iso_code_2'] = $this->url->link('localisation/country', 'user_token=' . $this->session->data['user_token'] . '&sort=iso_code_2' . $url);
 		$data['sort_iso_code_3'] = $this->url->link('localisation/country', 'user_token=' . $this->session->data['user_token'] . '&sort=iso_code_3' . $url);
+		$data['sort_status'] = $this->url->link('localisation/country', 'user_token=' . $this->session->data['user_token'] . '&sort=status' . $url, true);
```

Comment:
This line won't work in the master branch: https://github.com/opencart/opencart/pull/8225/commits/a33476ab80cc10a7ec609360894fbce9b221e220#diff-ae87ef052a1157e79231def36b367e57R292 .